src/ens/resolvers/index.ts

```diff
@@ -0,0 +1,57 @@
+import { GraphQLScalarType, Kind } from 'graphql';
```

Comment:
Commit https://github.com/ConsenSys/ethql/commit/5ba9227ead5d34e07c99ad98bfb673ec92c4296f introduced a change that allows the resolver map to be a function, taking in the previous resolvers and returning new ones that will be merged. Using this approach, `ens` wouldn't need to hard-reference the resolvers from core, it would just assume they are there (given the ordering sequence, as core is applied before), and would wrap around them accordingly. The way I see it, ens resolvers resolve the address promise, and call the original resolver that comes in the `prev` argument.